.github/actions/slack_notifier/action.yml

```diff
@@ -0,0 +1,52 @@
+name: Slack Notifier
+description: Send Slack updates to the provided channel upon workflow completion.
+
+inputs:
+  slack-bot-token:
```

Comment:
Can we please rename inputs to be snake case to mimic what we do elsewhere?

Reply:
done